src/program/render.rs

```diff
@@ -384,3 +384,26 @@ fn help_text() -> Vec<&'static str> {
         "+------------+----------------------------+-----------------------------------------------------+",
     ]
 }
+
+fn command_help_text() -> Vec<&'static str> {
+    vec![
+        "+-------+------------+----------+-------------------------------------+",
+        "| Short | Long       | Argument | Action                              |",
+        "+-------+------------+----------+-------------------------------------+",
+        "| ng    | newglob    | Required | The new glob/directory/file         |",
+        "| ?     | help       | None     | Toggle help box                     |",
```

Comment:
Should pressing `?` with the command mode help box open, but back in Normal mode, open up the Normal mode help box, or just close the Command mode box?

It feels nice, but unexpected at first.

Steps:
1. Open Command mode
2. Enter `?` as a command

Command help box is now open.

3. The use is back in Normal mode.
4. Press ?.
The Normal mode help box is displayed.

5. Press ? again
The normal mode help box is gone.

No more help boxes are visible.

Reply:
Yes, for me this was the intended behaviour and I'm happy with it.

Reply:
I'm happy with it as well. Just making sure.